tests/framework/CodeInterfaceTests/test_generic_interface.xml

```diff
@@ -27,6 +27,7 @@
     <Input name="one.xml" type="">one.xml</Input>
     <Input name="inp.two" type="">inp.two</Input>
     <Input name="mesh" type="">dummy.e</Input>
+    <Input name="a_dummy_file_for_subdirectory" type="" subDirectory="testSubDirectory">dummy_file_for_subdirectory.dummy</Input>
```

Comment:
The Inputs there are perturb-able  by the code interface. If we create a symbolic link we are going to perturb the original ones and we do not know what we are actually running.

Now if your suggestion is to create another option to specify if some files can be just "pointed" and not copied, I agree but it should be addressed in another PR/Issue